### ISSUE TITLE
[Yarn/Debugger localhost:7999 ECONNREFUSED]

### ISSUE THREAD

**Issue by twhite96** [devtools-html/debugger.html#7020]
When I tried to run the debugger the first time, the web socket kept disconnecting as referenced in #1069. I fixed in the `development.json` file by adding setting websockets to `true`.  However the examples on port 7999 have a: 

```
This site can’t be reached
localhost refused to connect.
Try:

Checking the connection
Checking the proxy and the firewall
ERR_CONNECTION_REFUSED
```

I have quit all Chrome and FIrefox instances. Trying to use DevTools CLI to listen on port 6080 will not work as the combination of `shift+f2` won't work: I'm on my Mac and those function keys are wonky.

Any ideas?

**Comment by jasonLaster** [comment]
CC @darkwing

**Comment by twhite96** [comment]
Any progress on this? I can't really run anything to help with this because I can't load the examples. It takes me to the DevTools url and when I check that tab in the debugger, there are no files available. When I put in the example port, the connection is refused.

**Comment by twhite96** [comment]
I am going to change the example port to see if that helps.

**Comment by twhite96** [comment]
Changing the port doesn't work. Followed the debugger directions on the DevTools website. Still having an issue.

**Comment by twhite96** [comment]
Looks like there is an issue with selenium web driver:

```
yarn run firefox
yarn run v1.6.0
(node:29807) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
$ start-firefox --start --location https://devtools-html.github.io/debugger-examples/
/Users/tiffanywhite/Desktop/Git_Repositories/debugger.html/node_modules/selenium-webdriver/lib/promise.js:2634
        throw error;
        ^

WebDriverError: connection refused
    at Object.throwDecodedError (/Users/tiffanywhite/Desktop/Git_Repositories/debugger.html/node_modules/selenium-webdriver/lib/error.js:489:11)
    at parseHttpResponse (/Users/tiffanywhite/Desktop/Git_Repositories/debugger.html/node_modules/selenium-webdriver/lib/http.js:513:13)
    at doSend.then.response (/Users/tiffanywhite/Desktop/Git_Repositories/debugger.html/node_modules/selenium-webdriver/lib/http.js:432:13)
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: WebDriver.createSession()
    at Function.createSession (/Users/tiffanywhite/Desktop/Git_Repositories/debugger.html/node_modules/selenium-webdriver/lib/webdriver.js:777:24)
    at Function.createSession (/Users/tiffanywhite/Desktop/Git_Repositories/debugger.html/node_modules/selenium-webdriver/firefox/index.js:667:55)
    at createDriver (/Users/tiffanywhite/Desktop/Git_Repositories/debugger.html/node_modules/selenium-webdriver/index.js:167:33)
    at Builder.build (/Users/tiffanywhite/Desktop/Git_Repositories/debugger.html/node_modules/selenium-webdriver/index.js:642:16)
    at start (/Users/tiffanywhite/Desktop/Git_Repositories/debugger.html/node_modules/devtools-launchpad/bin/firefox-driver.js:91:6)
    at Object.<anonymous> (/Users/tiffanywhite/Desktop/Git_Repositories/debugger.html/node_modules/devtools-launchpad/bin/firefox-driver.js:103:3)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)

    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I can't seem to run Firefox with `yarn run firefox` and the connection refused error still exists. I'll continue to investigate.

**Comment by jsomedon** [comment]
I am seeing exact same error about `ECONNREFUSED 127.0.0.1:6080` and `WebDriverError: connection refused` upon running after initial setup.

**Comment by kenjiO** [comment]
@jsomedon: For the Webdriver error see the comments in #6802 for a work around to avoid this crash.

**Comment by jasonLaster** [comment]
Just landed #7182, which should fix it.
